### DISCUSSION TITLE
What is the default PATH if WiX is installed using DotNet

### DISCUSSION THREAD

**Discussion by DatBytez** [wixtoolset/issues#8206] [Questions]
What is the correct Environmental Variables path for WiX if installed using DotNet.

It does not show up in my Program Files or my Program Files (x86)

It does appear in my C:\Users\Ted\.dotnet\tools\.store\wix\5.0.0\wix\5.0.0\tools\net6.0\any, but I have no clue which of those folders to actually point it to. I can check the wix --version and get 5.0.0+41e11442 and I can see it listed under dotnet tool list -g but when I try to run a jpackage --input . --dest . --main-jar MyJar.jar --main-class main.Main I get...
 
[10:35:12.117] Can not find WiX tools (light.exe, candle.exe)
[10:35:12.117] Download WiX 3.0 or later from https://wixtoolset.org and add it to the PATH.
Error: Invalid or unsupported type: [exe]

I have been trying for hours and just can't figure out how to get this to work.  Why does a project specifically made to help make software deployment easier not have an .exe to help deploy their software?

**Comment by robmen** [comment]
> Why does a project specifically made to help make software deployment easier not have an .exe to help deploy their software?

This comes across as fairly passive-aggressive. You should not expect a good reaction when you take your frustration out on us, especially when the issue is in the _other_ tool you are using.
